oauthenticator/google.py

```diff
@@ -53,6 +53,11 @@ def _scope_default(self):
 
     @gen.coroutine
     def authenticate(self, handler, data=None):
+        if not self.hosted_domain:
```

Comment:
Not requiring a hosted domain to be configured was a feature I think

Reply:
My logic was you should not allow any domain to login if you setup Google Oauth.

Reply:
I find it convenient that I can use this to require you have a "google" account but I don't actually care you have a particular domain. For example for workshops or other open events where people have all sorts of "google" accounts.